README.md

```diff
@@ -12,6 +12,17 @@ The data is available at the [CEDA Archive](https://data.ceda.ac.uk/badc/ar6_wg1
 under a Creative Commons CC-BY license, but the formats are quite diverse
 and not easy to handle.
 
+### Compiling the data
```

Comment:
My idea was to first have a high-level overview, then dive into the weeds - so maybe move this out of "Overview" and into an own section later?

Reply:
👍